linode/manager
# refactor: [M3-7436] - Update Account Queries to use Query Key Factory

## Description 📝

- Updates **account** related queries use a query key factory 🏭 
- Cleans up file/folder structure for account queries 📁 🧹 

## Preview 📷

> [!note]
> There should be no UI changes

## How to test 🧪

- Test general **account** related functionality 👤 
- Open the React Query dev tools and ensure query keys look correct 🔑 
- Test parent/child features and verify this does not break any of it 👪 

## As an Author I have considered 🤔
